apps/store/.env.example

```
@@ -7,3 +7,4 @@ VITE_SUPABASE_ANON_KEY=
 VITE_IMAGEKIT_PUBLIC_KEY=
 VITE_IMAGEKIT_URL_ENDPOINT=
 VITE_TOSS_CLIENT_KEY=
+VITE_GA_ID=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_GA_ID key should go before the VITE_IMAGEKIT_PUBLIC_KEY key

(UnorderedKey)